analyzer/src/main/kotlin/managers/Bundler.kt

```diff
@@ -146,11 +146,14 @@ class Bundler(
 
         options[OPTION_BUNDLER_VERSION]?.let { bundlerVersion ->
             val duration = measureTime {
-                org.jruby.Main().run(
-                    arrayOf(
-                        "-S", "gem", "install", "--no-document", "--user-install", "$BUNDLER_GEM_NAME:$bundlerVersion"
-                    )
-                )
+                val output = runScriptCode("""
+                    require 'rubygems/commands/install_command'
+                    cmd = Gem::Commands::InstallCommand.new
+                    cmd.handle_options ["--no-document", "--user-install", "$BUNDLER_GEM_NAME:$bundlerVersion"]
+                    cmd.execute
+                """.trimIndent()).trim()
```

Comment:
Commit-msg: Please provide a rationale "why" this is done.

Reply:
I've reworded the commit message a bit to hopefully make the rationale clearer.